ci-operator/step-registry/image-based-install-operator/reinstall/create-secrets/image-based-install-operator-reinstall-create-secrets-commands.sh

```diff
@@ -24,10 +24,33 @@ source network.sh
 oc create namespace ibi-cluster --dry-run=client -oyaml | oc apply -f -
 
 # Note: this must match the kubeconfig in the verify reinstall step
+kubeconfig=$(cat << 'EOKUBECONFIG'
```

Comment:
Hi, I just got `1 internal secret incidents detected! since I rebased the latest master branch. So, is it a fake kubeconfig here? If not, I guess you need to delete it and contact to the Security team, thanks!

Reply:
Hi @jianzhangbjz,  thanks for pointing this out, we've already submitted everything required to the Security team.